lambdas/service/app.py

```diff
@@ -187,8 +187,133 @@ def version():
     }
 
 
+def validate_repository_search(incoming_params, **validators):
```

Comment:
```suggestion
def validate_repository_search(params, **validators):
```

Typically, parameters are by definition "incoming". Distinguishing between incoming and outgoing parameters would require very special circumstances, that don't apply here. This is run-of-the-mill validation.

For that reason, please remove all other mentions of that word from this PR.